python/paddle/fluid/tests/unittests/mkldnn/test_elementwise_mul_mkldnn_op.py

```diff
@@ -51,13 +52,17 @@ def init_input_output(self):
     def test_check_grad_normal(self):
         pass
 
-    def test_check_grad_ingore_x(self):
-        pass
-
     def test_check_grad_ingore_y(self):
         pass
```

Comment:
Very nice catch. Reason is that there maybe something broken for reference implementation of test_check_grad_ingore_y , because it failed for that UT even without oneDNN being used. So I enabled only this test that I checked that works and left other commented out.